bool valid
builtin_interfaces/Time timestamp
float32 voltage_v
float32 current_a
float32 remaining
uint8 warning
